packages/xpath/test/xforms/randomize.test.ts

```diff
@@ -56,22 +56,18 @@ describe('randomize()', () => {
 			);
 		});
 
-		it(
-			'without a seed',
-			() => {
-				const expression = `randomize(${SELECTOR})`;
+		it('without a seed', { retry: 5 }, () => {
```

Comment:
This is the same test; the difference is that `{ retry: 5 }` needs to be passed as the second parameter instead of the last like before.